### DISCUSSION TITLE
get: products/ 관련 요청사항

### DISCUSSION THREAD

**Discussion by sh981013s** [bunderLikeLion/sellon_backend#101] [요청사항]
![image](https://user-images.githubusercontent.com/81363031/182770107-2c6beff8-ce85-4cd8-8ec2-129c88837fd8.png)

현재 위와같이 한줄에 3개의 아이템씩 배치하고 있는 상황이고

![image](https://user-images.githubusercontent.com/81363031/182770220-e78cfbd4-d1c1-46ee-a710-008fa13bc710.png)

`per_page` 가 10으로 셋팅되어 있는 상황이라 쪼끔 애매하게 출력이 되네요 ㅎㅎ
혹시 9로 수정이 가능할까여??

-----------------------------------------------------------------

![image](https://user-images.githubusercontent.com/81363031/182770382-045f946b-6a38-4edd-b12f-c0932587f4ae.png)

또한 총 개수도 출력해야 하는 상황이라 가능하다면 총 개수도 같이 담아서 보내주실 수 있으실까여??

**Comment by shinkeonkim** [comment]
@sh981013s 사실 개수 수정이 급하실때는 per_page=9 로 parameter보내주시면 됩니당 ㅎㅎ
일단 수정PR 올라와있습니다
